docs/guides.md

```diff
@@ -484,6 +500,546 @@ and refresh rate prior to connecting to an app. See
+issue similar to what Eric Dong found in their guide where SSH connections are not treated the same as graphical
+logins. This causes weirdness like sound not working through pipewire, and the tray icon for sunshine not appearing.
+To get around this, we need to close the SSH initiated sunshine service, and start a new sunshine service with the
+permissions of the graphical desktop. Unfortunately, this closes the connection and forces you to reconnect through
+moonlight. There is no way around this to the best of my knowledge without enabling auto-login.
+2. This guide does not cover using virtual displays. If you are using Nvidia graphics, check out Eric Dong's guide on
+Remote SSH Headless setups. If you are using AMD hardware, let me know if you find something or feel free to add it to
+this guide.
+3. I haven't (yet) found a way to disable sleep on the login screen, so if you wait too long after starting your PC,
+the display may go to sleep and moonlight will have trouble connecting. Shutting down and using WOL works great
+though.
+
+@attention{This is definitely safer than enabling auto-login directly, especially for a dual-use PC that is not only
+streamed via moonlight, but is also used as a standard desktop. *However* I do not know the implications of having an
+always running SSH client to the localhost on the same machine. It may be possible for someone with significant
+knowledge and physical access to the machine to compromise your user account due to this always-running SSH session.
+However, that's better than just having the desktop always available, or opening up SSH even just your LAN since this
+guide specifically disables non-localhost connections, so I believe this is safer to use than auto-login for general
+users. As always, your [threat model](https://en.wikipedia.org/wiki/Threat_model) may vary.}
+
+#### Prerequisites
+Go follow Eric Dong's guide for Remote SSH Headless Setup. Specifically these sections need to be completed and
+they've written an excellent guide which I do not feel the need to rehash here.
+
+[Static IP Setup](https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/guides/linux/headless_ssh.html#static-ip-setup)
+[SSH Server Setup](https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/guides/linux/headless_ssh.html#ssh-server-setup)
+[Virtual Display Setup](https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/guides/linux/headless_ssh.html#virtual-display-setup) (if desired and running an Nvidia GPU)
+[Uinput Permissions Workaround](https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/guides/linux/headless_ssh.html#uinput-permissions-workaround) (needed for keyboard/mouse to work on the login screen)
+[Stream Launcher Script](https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/guides/linux/headless_ssh.html#stream-launcher-script) (Will be called by our SSH client to start sunshine)
```

Comment:
```suggestion
1. [Static IP Setup](#static-ip-setup)
2. [SSH Server Setup](#ssh-server-setup)
3. [Virtual Display Setup](#virtual-display-setup)
4. [Uinput Permissions Workaround](#uinput-permissions-workaround)
5. [Stream Launcher Script](#stream-launcher-script)
```

These shorter versions will work no matter which version of the docs someone is viewing.